sdk/core/azure-core-test/pom.xml

```diff
@@ -105,6 +105,16 @@
       <version>4.5.1</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.projectreactor.netty</groupId>
+      <artifactId>reactor-netty-http</artifactId>
+      <version>1.0.27</version>  <!-- {x-version-update;io.projectreactor.netty:reactor-netty-http;external_dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-compress</artifactId>
+      <version>1.21</version>  <!-- {x-version-update;org.apache.commons:commons-compress;external_dependency} -->
```

Comment:
I recommend using the latest version of `commons-compress`

Reply:
@samvaity I have this one addressed locally, will push it tonight.